protocol/app/app.go

```diff
@@ -1288,16 +1288,14 @@ func New(
 		perpetualsmoduletypes.ModuleName,
 		statsmoduletypes.ModuleName,
 		satypes.ModuleName,
-		// should be before clob EndBlocker so that vault order cancels are
-		// processed before any vault order expirations (handled by clob)
-		vaultmoduletypes.ModuleName,
 		clobmoduletypes.ModuleName,
 		sendingmoduletypes.ModuleName,
 		vestmoduletypes.ModuleName,
 		rewardsmoduletypes.ModuleName,
 		epochsmoduletypes.ModuleName,
 		govplusmoduletypes.ModuleName,
 		delaymsgmoduletypes.ModuleName,
+		vaultmoduletypes.ModuleName,
```

Comment:
previously, vault endblocker was moved to be in front of clob to cancel orders before they expire to avoid flickering (as order expiration was set to be a very small value). With this PR, moving vault endblocker back as order expiration is significantly increased

Reply:
Why does the endblocker for the vault module need to be moved back?

Reply:
if vault endblocker is before clob's:
let's say an order is due for expiration, vault endblocker will run and see that the order still exists (thus don't do anything if price / size don't need updates), then clob endblocker will remove expired order. Then there's no order anymore until the next block's endblocker

Reply:
Note this behavior in a comment.